action.yml

```diff
@@ -34,6 +34,13 @@ inputs:
     required: false
     default: https://github.com/zephyrproject-rtos/sdk-ng/releases/download
 
+  skip-dependencies:
```

Comment:
i think 'skip-dependencies' is a bit too generic here, what dependencies exactly? we are talking about packages here, so maybe it should be explicit.

Maybe we could use `package-group` which can be set to none, but can also take other values, i.e. minimal, full, etc. with minimal including what we have now, full with more packages for special ops, like if we want coverage tools and such.